### ISSUE TITLE
Output not changing and GPU not being used

### ISSUE THREAD

**Issue by cbelth** [kuz/caffe-with-spearmint#12]
I specified to use the GPU in my solver.prototxt, but it is not being used. Also, I am getting the same output over and over again, and I'm not sure how to interpret it. I'm only fine tuning the learning rate and weight decay for now just to see how it works. Here is my output:

Submitted job 14 with local scheduler (process id: 27649).
Status: 1 pending, 0 complete.

Getting suggestion...

Suggestion:     NAME          TYPE       VALUE
                ----          ----       -----
                weight_decay  float      0.000100
                base_lr weig  float      0.000100
Submitted job 15 with local scheduler (process id: 27691).
Status: 1 pending, 0 complete.

Getting suggestion...

Suggestion:     NAME          TYPE       VALUE
                ----          ----       -----
                weight_decay  float      0.000100
                base_lr weig  float      0.000100
Submitted job 16 with local scheduler (process id: 27724).
Status: 1 pending, 0 complete.

Getting suggestion...

Suggestion:     NAME          TYPE       VALUE
                ----          ----       -----
                weight_decay  float      0.000100
                base_lr weig  float      0.000100
Submitted job 17 with local scheduler (process id: 27766).
Status: 1 pending, 0 complete.

Getting suggestion...

Suggestion:     NAME          TYPE       VALUE
                ----          ----       -----
                weight_decay  float      0.000100
                base_lr weig  float      0.000100
Submitted job 18 with local scheduler (process id: 27799).
Status: 1 pending, 0 complete.

Getting suggestion...

Suggestion:     NAME          TYPE       VALUE
                ----          ----       -----
                weight_decay  float      0.000100
                base_lr weig  float      0.000100
Submitted job 19 with local scheduler (process id: 27841).
Status: 1 pending, 0 complete.

**Comment by cbelth** [comment]
@kuz

**Comment by kuz** [comment]
Receiving the same suggestion means that Spearmint is not able to evaluate this set of paramteres -- there is an error happening somewhere. This would also explain why GPU is not used. Check the log files at `experiments/mnist/spearmint/output/*.out` (see the notes here: https://github.com/kuz/caffe-with-spearmint/tree/master/experiments/mnist#output).

**Comment by cbelth** [comment]
Thanks for the response. It turned out the lmdb package wasn't installed. Now that I have installed it, however, the "result" object returned in python_launcher in launcher.py of Spearmint is None (i.e. not an object). This is the case both in the mnist example and for my own network. Any ideas? @kuz

**Comment by cbelth** [comment]
I figured it out. I was able to trace the None object all the way back to 
caffe-with-spearmint/cwsm/cafferun.py where caffe_return_code was not 0. Caffe was getting a bus error because something had corrupted my train_lmdb directory. When I fixed that directory, it started to work.
